docs/http.rst

```diff
@@ -40,19 +41,21 @@ Lexicon
    :{UAID}: The Push User Agent Registration ID
 
 Push assigns each remote recipient a unique identifier. {UAID}s are UUIDs in
-lower case, dashed format. (e.g. '01234567-abcd-abcd-abcd-01234567abcd') This
+lower case, undashed format. (e.g. '01234567abcdabcdabcd01234567abcd') This
 value is assigned during **Registration**
 
    :{CHID}: The :term:`Channel` Subscription ID
 
 Push assigns a unique identifier for each subscription for a given {UAID}.
-Like {UAID}s, {CHID}s are UUIDs in lower case, dashed format.
-This value is assigned during **Channel Subscription**
+Like {UAID}s, {CHID}s are UUIDs in lower case, dashed format( e.g.
```

Comment:
well, sorta like. Either "Unlike {UAID}s" or "Like {UAIDS}s.. are UUIDs in lower case, *but* dashed"